.readthedocs.yaml

```diff
@@ -17,7 +17,7 @@ formats:
 build:
   os: ubuntu-22.04
   tools:
-    python: "3"
+    python: "3.8"  # We still support Python 3.8 for now.
```

Comment:
right. I'm targeting 3.8 for now because I assume there are canonical projects that are still using it, so using this here gives us another failsafe in case we add something to the extension that isn't 3.8-compatible.

Reply:
Thanks, that's what I thought but just wanted to make 100% sure :-)